server/db/scripts/old-records.csv

```diff
@@ -0,0 +1,691 @@
+id
```

Comment:
This is barely a CSV. We could format this to JS to avoid another dependency.

Reply:
True. PR updated.